packages/main/test/pages/Label.html

```diff
@@ -24,6 +24,7 @@
 </head>
 
 <body class="label1auto">
+	<div id="message"><ui5-message-strip design="Information">The proper use of the component is to define a label for elements like input, textarea, select.</ui5-message-strip></div>
```

Comment:
No-one will read these pages. These test files are not part of our documentation